packages/react-ui/src/widgets/legend/LegendProportion.js

```diff
@@ -1,43 +1,43 @@
-import React from 'react';
-import { Box, Grid } from '@mui/material';
-import makeStyles from '@mui/styles/makeStyles';
+import { Box, Grid, styled } from '@mui/material';
 import PropTypes from 'prop-types';
+import React from 'react';
 import Typography from '../../components/atoms/Typography';
 
-const useStyles = makeStyles((theme) => ({
-  circles: {
-    position: 'relative',
-    marginBottom: 4
-  },
-  avg: {
-    width: 12,
-    height: 2,
+const Circle = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'index'
+})(({ index = 0, theme }) => {
+  const sizes = {
+    0: 96,
+    1: 72,
+    2: 48,
+    3: 24
+  };
+  const width = `${sizes[index]}px`;
+  const height = `${sizes[index]}px`;
```

Comment:
You could use `theme.spacing()` also here.

Take a look at this component to see an example: `packages/react-ui/src/components/molecules/Avatar.js`